xds/src/main/java/io/grpc/xds/internal/rbac/engine/GrpcAuthorizationEngine.java

```diff
@@ -330,9 +343,13 @@ private int getDestinationPort() {
       SocketAddress addr = serverCall.getAttributes().get(Grpc.TRANSPORT_ATTR_LOCAL_ADDR);
       return addr == null ? -1 : ((InetSocketAddress) addr).getPort();
     }
+
+    private String getRequestedServerName() {
+      return "";
+    }
```

Comment:
Why introducing a private method for a constant value? Will it be implemented differently in the future?

Reply:
hm, was just doing https://github.com/grpc/proposal/pull/237/files#diff-129546e743e8d76e84156dd48fefe429eb95392c2954da44f02ed33642c95c70R99 no brainer. 
It was not like this in some other docs.
But this is kinda worrisome because it can match anything. 
cc @ejona86  Does the grpc `hardcode empty string` mean this or not?

Reply:
> But this is kinda worrisome because it can match anything.

This isn't a matcher, so it only matches if the matcher is okay with empty string.

> Does the grpc hardcode empty string mean this or not?

Yeah, it does. I think @dapengzhang0 is just surprised it added to an API surface instead of hard-coded in RequestedServerNameMatcher and wondering if that's because it is "coming soon."

Reply:
> > But this is kinda worrisome because it can match anything.
> 
> This isn't a matcher, so it only matches if the matcher is okay with empty string.

Yes. I meant that in the case that empty string is allowed, then it matches every request.

> > Does the grpc hardcode empty string mean this or not?
> 
> Yeah, it does. I think @dapengzhang0 is just surprised it added to an API surface instead of hard-coded in RequestedServerNameMatcher and wondering if that's because it is "coming soon."

It can also be hardcoded in RequestedServerNameMatcher, I put it here in API because this way it is more explicit about our mapping of all the fields.